lib/evaluateFunctions.js

```diff
@@ -96,4 +99,389 @@ function absoluteValue(node) {
+    radicandNode.args[1].content : radicandNode.args[1];
+  if (rootNode.equals(exponentNode)) {
+    newNode = baseNode;
+    return NodeStatus.nodeChanged(
+      MathChangeTypes.CANCEL_EXPONENT_AND_ROOT, node, newNode);
+  }
+  else if (NodeType.isConstant(rootNode) && NodeType.isConstant(exponentNode)) {
+    const rootValue = parseFloat(rootNode.value);
+    const exponentValue = parseFloat(exponentNode.value);
+    if (rootValue % exponentValue === 0) {
+      const newRootValue = rootValue/exponentValue;
+      const newRootNode = NodeCreator.constant(newRootValue);
+
+      newNode = NodeCreator.nthRoot(baseNode, newRootNode);
+      return NodeStatus.nodeChanged(
+        MathChangeTypes.CANCEL_EXPONENT, node, newNode);
+    }
+    else if (exponentValue % rootValue === 0) {
+      const newExponentValue = exponentValue/rootValue;
+      const newExponentNode = NodeCreator.constant(newExponentValue);
+
+      newNode = NodeCreator.operator('^', [baseNode, newExponentNode]);
+      return NodeStatus.nodeChanged(
+        MathChangeTypes.CANCEL_ROOT, node, newNode);
+    }
+  }
+
+  return NodeStatus.noChange(node);
+}
+
+// Returns the nthRoot evaluated for a multiplication node.
+// Expects a multiplication node.
+function nthRootMultiplication(node) {
+  let newNode = clone(node, false);
+  const rootNode = node.args.length === 2 ? node.args[1] : NodeCreator.constant(2);
+
+  let substeps = [];
+  let status;
+  if (NodeType.isConstant(rootNode) && !Negative.isNegative(rootNode)) {
+    status = factorChildren(newNode);
+    if (status.hasChanged()) {
+      substeps.push(status);
+      newNode = NodeStatus.resetChangeGroups(status.newNode);
+    }
+
+    status = groupTermsByRoot(newNode);
+    if (status.hasChanged()) {
+      substeps.push(status);
+      newNode = NodeStatus.resetChangeGroups(status.newNode);
+    }
+
+    status = convertMultiplicationToExponent(newNode);
+    if (status.hasChanged()) {
+      substeps.push(status);
+      newNode = NodeStatus.resetChangeGroups(status.newNode);
+      if (newNode.args[0].op === '^') {
+        status = nthRootExponent(newNode);
+        substeps.push(status);
+        return NodeStatus.nodeChanged(
+          MathChangeTypes.NTH_ROOT_VALUE, node, status.newNode, true, substeps);
+      }
+    }
+  }
+
+  status = distributeNthRoot(newNode);
+  if (status.hasChanged()) {
+    substeps.push(status);
+    newNode = NodeStatus.resetChangeGroups(status.newNode);
+  }
+
+  status = evaluateNthRootForChildren(newNode);
+  if (status.hasChanged()) {
+    substeps.push(status);
+    newNode = NodeStatus.resetChangeGroups(status.newNode);
+
+    status = combineRoots(newNode);
+    if (status.hasChanged()) {
+      substeps.push(status);
+      newNode = NodeStatus.resetChangeGroups(status.newNode);
+    }
+
+    return NodeStatus.nodeChanged(
+      MathChangeTypes.NTH_ROOT_VALUE, node, newNode, true, substeps);
+  }
+
+  return NodeStatus.noChange(node);
```

Comment:
ohh okay so you put it back together in combineRoots, but that's not always called after the distribute right?